image_layers/query-map-image-sublayer/src/main/java/com/esri/samples/query_map_image_sublayer/QueryMapImageSublayerSample.java

```diff
@@ -72,20 +71,24 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a map with a basemap and set its initial viewpoint
-      ArcGISMap map = new ArcGISMap(Basemap.createStreetsVector());
-      Point initialLocation = new Point(-12716000.00, 4170400.00, SpatialReferences.getWebMercator());
-      Viewpoint viewpoint = new Viewpoint(initialLocation, 6000000);
-      map.setInitialViewpoint(viewpoint);
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with a basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
 
       // create and add a map image layer to the map
       ArcGISMapImageLayer imageLayer = new ArcGISMapImageLayer("https://sampleserver6.arcgisonline.com/arcgis/rest/services/USA/MapServer");
       map.getOperationalLayers().add(imageLayer);
 
-      // create a map view and set the map to it
+      // create a map view and set its map
       mapView = new MapView();
       mapView.setMap(map);
 
+      // set a viewpoint on the map view
+      mapView.setViewpoint(new Viewpoint(36.75056, -115.44154, 9000000));
```

Comment:
was there a reason this scale changed to 9000000 when the original scale was set to 6000000?

Reply:
No.. corrected